Frontend/research/src/Reducer/Reducer.js

```diff
@@ -1,4 +1,28 @@
-import { SIGN_UP, SIGN_UP_SUCCESS, SIGN_UP_FAILURE, SIGN_IN, SIGN_IN_SUCCESS, SIGN_IN_FAILURE, SIGNED_IN, SIGNED_OUT, GET_LINKS, GET_LINKS_SUCCESS, GET_LINKS_FAILURE, GET_LINK, GET_LINK_SUCCESS, GET_LINK_FAILURE, ADD_LINK, ADD_LINK_SUCCESS, ADD_LINK_FAILURE, DELETE_LINK, DELETE_LINK_SUCCESS, DELETE_LINK_FAILURE } from '../Actions/Index';
+import {
```

Comment:
you can use a * here and say import * as actions from '../Actions'   then when you call them it would be actions.SIGN_UP for example so you don't have to write out every single one in the import